web/docs/Frontend Starters/lando-template-local-dev-backend-and-frontend.md

```diff
@@ -0,0 +1,80 @@
+# Instructions for Local Development of Decoupled Front End and CMS Backend using Lando
+
+## Prerequisites
+
+- Install [docker](https://docs.docker.com/get-docker/)
+- Install [lando](https://docs.lando.dev/getting-started/installation.html)
+- [Generate Pantheon Machine Token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
+
+## Local Development with Lando for -
+
+1. Your NextJS front end site
+
+- In your current directory create a new sub-directory e.g. "fe". Clone this front end repo into the "fe" sub-directory [next-drupal-starter](https://github.com/pantheon-systems/next-drupal-starter):
+        
+```
+mkdir fe;\
+> cd fe;\
+> git clone https://github.com/pantheon-systems/next-drupal-starter .
+```    
+
+- Create a lando file similar to the one below:
```

Comment:
This would need to be explicit about the file name - .lando.yml. Also if the instructions to clone above are followed there already is a lando file in the codebase. These instructions to create a lando file will likely be confusing without some additional context.

Reply:
Done and also changed the instruction sequence as mentioned in earlier comment